contracts/ERC721A.sol

```diff
@@ -270,6 +275,8 @@ contract ERC721A is IERC721A {
      */
     function _packOwnershipData(address owner, uint256 flags) private view returns (uint256 value) {
         assembly {
+            // Mask `owner` to the lower 160 bits, in case the upper bits somehow aren't clean.
+            owner := and(owner, 0xffffffffffffffffffffffffffffffffffffffff)
```

Comment:
I kinda like having `(1 << 160) - 1` as a top level constant instead of 40 f's. I know you like hex values for assembly though, so you can have the final call